src/App.js

```diff
@@ -3,30 +3,43 @@ import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import { ifDev } from "./utils/removeAttribute.js";
 
 // local imports
-// import Dashboard from "./components/dashboard/index.js";
-// import Login from './auth/Login';
-// import Register from './auth/Register';
-import LoadingScreen from "./components/layout/LoadingScreen.js";
-import WatchList from "./components/movies/WatchList.js";
+import PrivateRoute from "./utils/PrivateRoute.js";
+import Dashboardv1 from "./components/dashboard/Dashboardv1.js";
+import Navigation from "./components/dashboard/Navigation.js";
+import Register from "./components/auth/Register";
 
 // config imports
 import reactGAinitialization from "./config/analytics.js";
 
+// creating the store
+import { Provider } from "react-redux";
+import { createStore, compose, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import logger from "redux-logger";
+import { reducer } from "./store/reducers";
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const store = createStore(
+  reducer,
+  composeEnhancers(applyMiddleware(thunk, logger))
+);
+
 function App() {
   useEffect(() => reactGAinitialization(), []);
 
   return (
-    <Router>
-      <div className="App" data-test={ifDev("App-component")}>
-        <Switch>
-          {/* <Route exact path="/" component={Dashboard} /> */}
-          {/* <Route path='/login' component = {Login}/>
-      <Route path='/register' component = {Register}/> */}
-          <Route path="/watch-list" component={WatchList} />
-          <Route exact path="/" component={LoadingScreen} />
-        </Switch>
-      </div>
-    </Router>
+    <Provider store={store}>
+      <Router>
+        <div className="App" data-test={ifDev("App-component")}>
+          <Navigation />
```

Comment:
I added some functionality so the Navigation component will dynamically change.

Reply:
awesome!